packages/venia-ui/lib/components/AddressBookPage/addressCard.js

```diff
@@ -25,6 +25,10 @@ const AddressCard = props => {
 
     const classes = mergeClasses(defaultClasses, propClasses);
 
+    const handleEdit = useCallback(() => {
+        onEdit(address);
+    }, [address, onEdit]);
```

Comment:
Should I create a talon just for this?

Honestly I'd much prefer to pass a bound function here for this instead.

Reply:
I support moving it up to `addressBookPage` in the creation loop; just pass bound functions instead of doing it here. I don't mind this as-is, but there may be a good argument to keep this purely presentational.

Reply:
Great call, done in https://github.com/magento/pwa-studio/pull/2879/commits/180b344b7bf2d373c662ec2cc168e0fc6a3098b4.